cfn/paws-collector.template

```diff
@@ -347,10 +352,11 @@
                                 "\n",
                                 "exports.handler = (event, context, callback) => {\n",
                                 "    if (event.ResourceType == 'AWS::CloudFormation::CustomResource' &&\n",
-                                "        event.RequestType == 'Create') {\n",
+                                "        event.RequestType == 'Create' &&\n",
+                                "        event.ResourceProperties.Plaintext.length  > 0) {\n",
                                 "        return encrypt(event, context);\n",
                                 "    }\n",
-                                "    return response.send(event, context, response.SUCCESS);\n",
+                                "    return response.send(event, context, response.SUCCESS, {EncryptedText: \"\"});\n",
```

Comment:
Why do we need this?

Reply:
because that value is referenced later in the cfn template. It was throwinng an error if it wasn't being encrypted. i figured it was better to just return an empty string from this encrypt function than use a bunch of funky conditions in the cfn. You can encrypt an empy string with kms. It needs to have a minimum length of 1.